app/views/games/show.html.erb

```diff
@@ -1,10 +1,35 @@
-<p style="color: green"><%= notice %></p>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Game Information</title>
+</head>
+<body>
 
-<%= render @game %>
+<p style="color: green"><%= notice %></p>
 
-<div>
-  <%= link_to "Edit this game", edit_game_path(@game) %> |
-  <%= link_to "Back to games", games_path %>
+<h1>Game Information</h1>
 
-  <%= button_to "Destroy this game", @game, method: :delete, data: { turbo_confirm: "Are you sure to delete?", disable_with: "Processing..."} %>
+<div class="games-container">
+  <table>
+    <% @game.attributes.each do |attribute, value| %>
```

Comment:
This does not seem to discriminate which attributes are displayed. For general use we do not want 'Included' and 'Exclusion Notes' columns visible. There might be some others like that that should be prevented from beign displayed (unless you're an admin/moderator).